src/__tests__/get-validator_test.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import getValidator from "../get-validator.js";
+import * as ValidateAndJson from "../validate-and-json.js";
+import validateAndFix from "../validate-and-fix.js";
+import validateAndReport from "../validate-and-report.js";
+
+import type {Options} from "../types.js";
+
+describe("#getValidator", () => {
+    it("should return the result of getValidateAndJson(jsonItems) when options.json is true", () => {
+        const options: Options = {
```

Comment:
NIT: Could you ad the `// Arrange`, `// Act`, `// Assert` section headers to these test cases?